backend/app/main.py

```
@@ -0,0 +1,217 @@
+    """Health check endpoint."""
+    return {"status": "healthy"}
+
+@app.get("/readyz")
+async def readiness_check():
+    """Readiness check endpoint."""
+    return {"status": "ready"}
+
+@app.post("/api/translate")
+async def translate_document(
+    background_tasks: BackgroundTasks,
+    file: UploadFile = File(...),
```

Ruff (0.13.1)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)